examples/async_replication/async_replicated_container.h

```diff
@@ -0,0 +1,421 @@
+#pragma once
+
+#include "../../bricks/sync/waitable_atomic.h"
+#include "../../blocks/http/api.h"
+#include "../../bricks/time/chrono.h"
+
+#include <sstream>
+#include <queue>
+
+#define htonll(x) ((1==htonl(1)) ? (x) : (((uint64_t)htonl((x) & 0xFFFFFFFFUL)) << 32) | htonl((uint32_t)((x) >> 32)))
```

Comment:
This can definitely be `template`-d and `constexpr`-d, so that the check is not performed on every call to them, and the call is inlined too.

Reply:
done. Found `htobe64` and `be64toh` function, removed this code